test/sanity/validate-modules/ignore.txt

```diff
@@ -444,10 +444,7 @@ lib/ansible/modules/cloud/azure/azure_rm_deployment.py E322
 lib/ansible/modules/cloud/azure/azure_rm_deployment.py E324
 lib/ansible/modules/cloud/azure/azure_rm_deployment.py E325
 lib/ansible/modules/cloud/azure/azure_rm_deployment.py E326
-lib/ansible/modules/cloud/azure/azure_rm_dnsrecordset.py E322
-lib/ansible/modules/cloud/azure/azure_rm_dnsrecordset.py E324
-lib/ansible/modules/cloud/azure/azure_rm_dnsrecordset.py E325
-lib/ansible/modules/cloud/azure/azure_rm_dnsrecordset.py E326
+lib/ansible/modules/cloud/azure/azure_rm_dnsrecordset.py E321
```

Comment:
PRs should not add ignores. The underlying issue needs to be addressed.

Reply:
So how can I address the issue? And the sanity test? I am really confused by this.

Reply:
I'd start by working down the list of import errors. The first one I see is:

> AttributeError: 'AzureRMRecordSet' object has no attribute '_dns_client'

Which is reported as being called from here:

https://github.com/ansible/ansible/pull/35823/files#diff-e448c30ab815c0d2e05412e35bc7da19R245

The issue appears to be that the base class constructor hasn't been called yet, so `_dns_client` hasn't been initialized yet:

https://github.com/VSChina/ansible/blob/367d66b8a67a568fe3a02dc6cee36f5c033728d3/lib/ansible/module_utils/azure_rm_common.py#L260

Reply:
You may find it easier to try fixing the integration test failures first:

https://app.shippable.com/github/ansible/ansible/runs/56819/62/tests
https://app.shippable.com/github/ansible/ansible/runs/56819/68/tests